### PR TITLE
build: add a linkinator step

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -93,3 +93,12 @@ jobs:
           node-version: 12
       - run: npm install
       - run: node ./.github/workflows/finale.js '${{needs.changeFinder.outputs.requiredJobs}}' ${{secrets.GITHUB_TOKEN}}
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: JustinBeckwith/linkinator-action@v1
+        with:
+          paths: '**/*.md'
```
Comment (bcoe) [inline .github/workflows/ci.yaml:102]: neat.

File: packages/conventional-commit-lint/README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ npm run test:snap
 
 If you have suggestions for how conventional-commit-lint could be improved, or want to report a bug, open an issue! We'd love all and any contributions.
 
-For more, check out the [Contributing Guide](CONTRIBUTING.md).
+For more, check out the [Contributing Guide](../../CONTRIBUTING.md).
```
Comment (bcoe) [inline packages/conventional-commit-lint/README.md:38]: see it's already catching things.

**Review by bcoe — APPROVED** [review]
Great addition 👍

**Comment by chingor13** [comment]
#1420 will fix the generate-bot tests
For the monitoring-system tests, it looks like the interrogate.js is confused because it assumes anything touched under monitoring-system is a directory.

**Review by chingor13 — APPROVED** [review]

**Pull request merged**